playbooks/appsemblerPlaybooks/ficus_enterprise.yml

```diff
@@ -96,3 +96,21 @@
     - { role: "xqueue", update_users: True }
     - role: "githubsshkeys"
       when: "cloud_provider == 'azure'"
+
+- name: Install Scampersand Reporting
+  hosts: "edxapp-server-0"
```

Comment:
@melvinsoft Thanks for clarifying this. Good to know (and might be worth putting an inline comment to this fact in the playbooks so in the future the readers know why these aren't grouped